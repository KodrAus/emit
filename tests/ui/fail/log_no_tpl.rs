extern crate emit;

fn main() {
    emit::info!(to: |_| {});
}
