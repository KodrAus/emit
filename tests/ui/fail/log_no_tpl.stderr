error: parsing failed: missing string template
 --> fail/log_no_tpl.rs:4:17
  |
4 |     emit::info!(to: |_| {});
  |                 ^^^^^^^^^^
